src/components/PostDetails.tsx

```diff
@@ -2,91 +2,47 @@ import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import {} from 'react-redux';
```

Comment:
The import statement for `react-redux` is empty. Ensure that any necessary hooks or components from `react-redux` are imported, or remove this line if it's not needed.